Bottom: a01c128ea7bf2cc6586c2696c46c1b4a22dcb52c
Top:    6a3ea35ed46e4cf08b635cf392766403e8d0d6ba
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 15:37:39 +0800

Refresh of autofs-5.0.7-allow-use-of-hosts-map-in-maps.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index ab77e58..87b93b4 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -1440,8 +1440,6 @@ int parse_mount(struct autofs_point *ap, const char *name,
 			p += l;
 			p = skipspace(p);
 
-			error(LOGOPT_ANY, "options %s", options);
-
 			l = parse_mapent(p, options, &myoptions, &loc, ap->logopt);
 			if (!l) {
 				if (!(strstr(myoptions, "fstype=autofs") &&
